person_class.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = nil
```

Comment:
- [ ] [OPTIONAL] Instead of having the `id` equal `nil` every time, you can do something like this for now:

```ruby
@id = Random.rand(1..1000)
```